### ISSUE TITLE
Peer review

### ISSUE THREAD

**Issue by RobBlumberg** [UBC-MDS/DSCI-532-Group108#76]
Hi Team 108,

My peer review feedback will be based on the principles of effective dashboard design / effective visualizations. 

Aesthetics: The general layout of the dashboard is clean and well organized. I liked how you  partitioned the different elements of the dashboard into different areas using containers. This helps the user focus on specific areas of the dashboard and the relevant information related to those areas. I also liked your use of colours throughout the dashboard. The colour choice for the scale clearly shows which areas had higher crime counts than others. Additionally, your use of blue / red in the plot comparing crime counts in specific neighbourhoods maps well with the drop down menu.

Allow users to filter data: You did a really good job at adhering to this principle. The first plot on the left allows me to filter for different types of crimes across all neighbourhoods in Vancouver, and I can also filter the crime index to compare neighbourhoods with lower crimes indices. This avoids neighbourhoods with large indices saturating the others, and so increases the effectiveness of your visualization. The second visualization adds an additional layer of filtering as I can compare specific neighbourhoods even more closely. In all, through your various interactive features, you allow users to dive deep into the data, without introducing too much complexity. 

Narrative and signposts: Your description of the crime index metric is important as it helps users understand the visualization. However, I thought your dashboard could have used a few additional signposts explaining how to use the different interactive elements. For example, a simple note containing a message such as "Select two neighbourhoods to compare crime counts" could have been useful to guide the user in their exploration of the dashboard. 

Purpose-driven dashboards: This point is related to the previous one about the narrative. It would have been useful if you provided a brief introduction explaining the specific purpose of the dashboard. Of course, analyzing crime counts by neighbourhood is important and could have a wide range of applications. Thus, if the dashboard was to be used for many applications, even simply stated this would have provided some context for the user. In any case, brief introduction paragraphs are generally worth adding. 

To summarize, you did a great job at visualizing the crime data using colours, a clean layout and effective filtering functionalities. Adding descriptions and signposts would have increased user experience even further.
